03-altalanos.md

```diff
@@ -51,7 +51,7 @@
 
 19.  Minden érvényes találat után, vagy ha az asszót más okból megállítják (zavart vívás, meghibásodás, sikertelen birkózási kísérlet, stb.), a mérkőzés a felállási vonalról folytatódik, vívóállásból.
 
-20.  Amennyiben a technikai feltételek adottak, úgy minden vívónak mérkőzésenként egyszer van lehetősége videó felülvizsgálatot kérni. Ha a videó felülvizsgálat az azt kérelmező vívó javára változtatja meg a bíró ítéletét, a kérelmező vívó továbbra is megtartja a videókérés lehetőségét.
+20.  Amennyiben a technikai feltételek adottak, úgy a csoport és az egyenes kieséses ágon is minden vívónak mérkőzésenként egyszer van lehetősége videó felülvizsgálatot kérni. A videó felülvizsgálat technikai feltételeit az egyenes kieséses ágon a szervezőknek biztosítani szükséges. Ha a videó felülvizsgálat az azt kérelmező vívó javára változtatja meg a bíró ítéletét, a kérelmező vívó továbbra is megtartja a videókérés lehetőségét.
```

Comment:
Akkor viszont az első félmondatot szedjük ki, mert ellentmond annak h szabály szerint biztosítani KHELL

Az egyenes kieséses ÁG helyett legyen mondjuk egyenes kieséses mérkőzések, mert hogy az ág az, amin egy vívó végighalad a verseny során ( amit a rangsor ad ki).

Reply:
Javítsuk így.

Reply:
Jó, mivel ez a te PR-ed, lécci írd bele és commitold fel, ahogy eddig is. 
Automatikusan meg fog jelenni a pull-requestben, azt már nem kell külön. Aztán bemergelem.